app/src/main/res/layout/item_album.xml

```diff
@@ -62,33 +77,40 @@
                 android:layout_marginTop="15dp"
                 android:textAppearance="?android:attr/textAppearanceLarge"
                 android:textColor="@color/piwigo_orange"
+                tools:text="peterdqdwqdqwdqdwqdqw"
                 app:layout_constraintTop_toTopOf="parent"
-                app:layout_constraintStart_toStartOf="@+id/guideline"
+                app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintEnd_toEndOf="parent"
                 />
 
             <TextView
                 android:id="@+id/title2"
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:text="@{viewModel.photos}"
-                android:singleLine="false"
                 android:gravity="center"
-                android:paddingBottom="8dp"
+                android:paddingBottom="20dp"
+                android:singleLine="false"
+                android:text="@{viewModel.photos}"
                 android:textAppearance="?android:attr/textAppearanceSmall"
                 android:textColor="@color/piwigo_dark_text_color"
                 app:layout_constraintBottom_toBottomOf="parent"
-                app:layout_constraintStart_toStartOf="@+id/guideline"
                 app:layout_constraintEnd_toEndOf="parent"
+                tools:text="textetxtextefwefwefwefwefwefwefwefwefwetx"
```

Comment:
Same thing here, "textview2" would do the trick ;)

Reply:
Again, best would be to choose the description of the content here.